examples/account/evmapp/src/main/java/io/horizen/examples/EvmAppModule.java

```diff
@@ -104,5 +105,9 @@ public void configureApp() {
         bind(Integer.class)
                 .annotatedWith(Names.named("ConsensusSecondsInSlot"))
                 .toInstance(consensusSecondsInSlot);
+
+        bind(String.class)
+                .annotatedWith(Names.named("EonVersion"))
+                .toInstance(eonVersion);
```

Comment:
The SDK is a generic framework to build sidechains, here we are adding a reference to a specific project.
I think the best way here is to bind the value to a generic "AppVersion".
So i would rename:

- "EonVersion" -> "AppVersion"
- eonVersion -> appVersion